### ISSUE TITLE
Restricing single line input fields to one line

### ISSUE THREAD

**Issue by richiesun** [one-signal/emoji-picker#10]
How do you keep the single input fields to stay on one line? It seems that you can break return and have multiline even though it's a single line input field. Makes sense for textareas, but should not happen for single line input fields.

**Comment by gdeglin** [comment]
Yes, single line inputs are not supported yet. If you end up modifying the library to add this feature, please make a pull request and we'll be sure to accept it.

**Comment by gdeglin** [comment]
Didn't mean to close this. Leaving open since it's a valid issue.

**Comment by jaxn** [comment]
This is particularly a concern for messaging apps where hitting return should submit the form.
